### PR TITLE
The solution of the 'car repository' problem

### PR DESCRIPTION
**Pull request by koryun23**

### CODE REVIEW

File: src/main/com/aca/homework/week8/car/repository/CarRepository.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.aca.homework.week8.car.repository;
+
+public class CarRepository {
+    private Car[] cars = new Car[1000];
+    private int numberOfCars = 0;
+
+    public Car save(Car car){
+        if(findByVin(car.getVin()) != null){
```
Comment (gevorgyanarman) [inline src/main/com/aca/homework/week8/car/repository/CarRepository.java:8]: the update is not implemented if the vin already exists

File: src/test/com/aca/homework/week8/car/repository/CarRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    @BeforeEach
+    public void setUp(){
+        testSubject = new CarRepository();
+    }
+
+    @Test
+    public void testSave(){
+        Car car = new Car("abcd1234", 2007, 3);
+        testSubject.save(car);
```
Comment (gevorgyanarman) [inline src/test/com/aca/homework/week8/car/repository/CarRepositoryTest.java:19]: just write testSubject.save(new Car("abcd1234", 2007, 3))

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

**Comment by koryun23** [comment]
done

File: src/main/com/aca/homework/week8/car/repository/Car.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    public void setYear(int year) {
+        this.year = year;
+    }
+
+    public int getOwnersCount() {
+        return ownersCount;
+    }
+
+    public void setOwnersCount(int ownersCount) {
```
Comment (gevorgyanarman) [inline src/main/com/aca/homework/week8/car/repository/Car.java:30]: someone can set a negative number here.

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]
please do the change and merge the pull request

**Pull request merged**